ValLib/auth/context.py

```diff
@@ -74,7 +74,7 @@ def riot_ssl_ctx() -> ssl.SSLContext:
 
     with warnings.catch_warnings():
```

Comment:
Lines related to this warning suppression and the comment should be removed with this change as only TLS v1 and v1.1 are currently deprecated since Python 3.10.